physionet-django/templates/about/software_content.html

```diff
@@ -1,1468 +1,15 @@
-<section id="citation">
-  <h1>Suggested Citation</h1>
-  <p>If you use resources from PhysioNet in a publication, please credit
-  the author(s) using the citation displayed at the top of the published content. Please also include the standard citation for PhysioNet:</p>
-
-  <p class="alert alert-primary" role="alert">Goldberger AL, Amaral LAN, Glass L,
-    Hausdorff JM, Ivanov PCh, Mark RG,
-  Mietus JE, Moody GB, Peng C-K, Stanley HE.  PhysioBank, PhysioToolkit, and
-  PhysioNet: Components of a New Research Resource for Complex Physiologic
-  Signals.
-  <i>Circulation</i> <b>101</b>(23):e215-e220 [Circulation Electronic Pages;
-  <a href="http://circ.ahajournals.org/content/101/23/e215.full"
-  target="other">http://circ.ahajournals.org/content/101/23/e215.full</a>];
-  2000 (June 13).</p>
-
+<section id="overview">
+    <h2>Overview</h2>
+  <p>This page displays an alphabetical list of all software projects on PhysioNet. To search content on PhysioNet, <a href="{% url 'content_index' %}">visit the search page</a>. Enter the search terms, add a filter for resource type if needed, and select how you would like the results to be ordered (for example, by relevance, by date, or by title).</p>
+  <p>Many of the time series datasets on PhysioNet (for example, ECGs) are shared in waveform-database (WFDB) compatible file formats.  For more information on these formats, see the <a href="https://www.physionet.org/physiotools/wag" />WFDB Applications Guide</a>. WFDB-formatted datasets can also be viewed online with <a href="{% url 'lightwave_home' %}">LightWAVE</a>.</p>
```

Comment:
please could you take another look at this URL? i think, at the least, relative would be better than absolute (more flexible, avoids breaking if the root changes).